plugins/masters-tournament/masters_renderer.py

```
@@ -152,6 +153,24 @@ def _load_font_sized(filename: str, size: int) -> Optional[ImageFont.ImageFont]:
 
 
+def _cleanup_bdf_temp() -> None:
+    """Remove the BDF temp directory on process exit."""
+    global _BDF_TEMP_DIR
+    if _BDF_TEMP_DIR is None:
+        return
+    try:
+        import shutil
+        shutil.rmtree(_BDF_TEMP_DIR, ignore_errors=True)
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)